scripts/quickstart.sh

```diff
@@ -130,6 +133,33 @@ function getAndUpdateBlockNumbers {
   sed -i "s/^FOREIGN_START_BLOCK.*/FOREIGN_START_BLOCK=${ETHBLOCK}/" "$ENV_FILE"
 }
 
+function checkEthGasAPI {
+  resetOracle=false
+
+  if [ -z "$FOREIGN_GAS_PRICE_ORACLE_URL" ] ; then
+    WARNINGS+=("No eth gas station api set please update your env file to include FOREIGN_GAS_PRICE_ORACLE_URL which should set an ethgasstation endpoint see https://data.defipulse.com/ for more details")
+  else
+    if [[ "$FOREIGN_GAS_PRICE_ORACLE_URL" == *"https://data-api.defipulse.com/api/v1/egs/api/ethgasAPI.json?api-key="* ]]; then
+      status_code=$(curl --write-out %{http_code} --silent --output /dev/null "$FOREIGN_GAS_PRICE_ORACLE_URL")
+
+      if [[ "$status_code" == 200 ]] ; then
+        echo "Positive response from gas oracle"
+      else
+        WARNINGS+=("trying to grab data from $FOREIGN_GAS_PRICE_ORACLE_URL is giving errors, using the default oracle")
+        resetOracle=true
+      fi
+    else
+      WARNINGS+=("FOREIGN_GAS_PRICE_ORACLE_URL Does not match ethgasstation endpoint see https://data.defipulse.com/ for more details, using the default oracle, recommend to create your own!")
+      resetOracle=true
+    fi
+
+    if [ "$resetOracle" = true ] ; then
+      echo "Reset FOREIGN_GAS_PRICE_ORACLE_URL back to default"
+      sed -i "s/^FOREIGN_GAS_PRICE_ORACLE_URL.*/FOREIGN_GAS_PRICE_ORACLE_URL=$DEFAULT_GAS_ORACLE/" "$ENV_FILE"
```

Comment:
@Andrew-Pohl I had a late idea on that, if you just put an empty string in  `FOREIGN_GAS_PRICE_ORACLE_URL`, it'll use the default value in the container, right? So we don't need to keep the oracle url in multiple places

Reply:
That's a very good point!, If it's incorrect we can just delete it from this env file and let the default do its job! 👍